extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsMiddleware/FunctionsHttpProxyingMiddleware.cs

```diff
@@ -105,6 +108,21 @@ private static Task<bool> TryHandleOutputBindingsHttpResult(FunctionContext cont
                 : TryHandleHttpResult(httpOutputBinding.Value, context, httpContext);
         }
 
+        private static Task<bool> TryClearHttpOutputBinding(FunctionContext context) 
```

Comment:
Should this method return a `Task<T>`? There is no asynchronous operation happening inside this method. correct?